### PR TITLE
Added QFX5210 readme for build and install

### PR DESCRIPTION
**Pull request by ashishb-juniper**
Documentation for installing SONiC on Juniper QFX5210 platform

Signed-off-by: Ciju Rajan K <crajank@juniper.net>
Signed-off-by: Ashish Bhensdadia <bashish@juniper.net>

### CODE REVIEW

**Comment by ashishb-juniper** [comment]
@lguohan @jleveque 

Please take a look pull request.

-Ashish

**Comment by ashishb-juniper** [comment]
@jleveque 

incorporated format related comments. Please take a look pull request.

Thanks
Ashish

**Comment by ashishb-juniper** [comment]
@jleveque 

Can you please take a look?

-Ashish

File: platform/broadcom/sonic-platform-modules-juniper/README.md [hunk 1]
```diff
@@ -1 +1,291 @@
...
+
+    Boot Option Priorities
+
+    Boot Option #1          [JetFlashTranscend 8...]
+
+    Boot Option #2          [ATP ATP IG eUSB 1100]
+
+    Boot Option #3          [IBA GE Slot 00A0 v1543]
+
+    Boot Option #4          [UEFI: Built-in EFI ...]
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-juniper/README.md]: Suggest to surround this in triple backticks to ensure it is displayed in monospaced font.

File: platform/broadcom/sonic-platform-modules-juniper/README.md [hunk 2]
```diff
@@ -1 +1,291 @@
...
+
+ |                                                                            |
+
+ +----------------------------------------------------------------------------+
+
+
+
+      Use the ^ and v keys to select which entry is highlighted.
+
+      Press enter to boot the selected OS, `e' to edit the commands before booting or `c' for a command-line.
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-juniper/README.md]: Surround this in triple backticks to ensure it is displayed in monospaced font.

File: platform/broadcom/sonic-platform-modules-juniper/README.md [hunk 3]
```diff
@@ -1 +1,291 @@
...
+ |                                                                            |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            |
+ |                                                                            | 
+ +----------------------------------------------------------------------------+
+
+      Use the ^ and v keys to select which entry is highlighted.          
+      Press enter to boot the selected OS, `e' to edit the commands       
+      before booting or `c' for a command-line.
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-juniper/README.md]: Surround this in triple backticks to ensure it is displayed in monospaced font.

Comment (ashishb-juniper) [reply platform/broadcom/sonic-platform-modules-juniper/README.md]: @jleveque 

Please take a look

File: platform/broadcom/sonic-platform-modules-juniper/README.md (hunk 2 above)
Comment (ashishb-juniper) [reply platform/broadcom/sonic-platform-modules-juniper/README.md]: done. Please take a look

Comment (jleveque) [reply platform/broadcom/sonic-platform-modules-juniper/README.md]: Thanks. However, this output appears to have extra blank lines every other line (lines 141, 143, 145, ...). Can you please remove them?

**Comment by ashishb-juniper** [comment]
@jleveque  deleted extra blank lines.

**Comment by ashishb-juniper** [comment]
@jleveque 
Why it is not merged with master. What can be done from my side. is vs and vsimage cause a issue?

**Comment by jleveque** [comment]
Retest this please

**Comment by jleveque** [comment]
@ashishb-juniper: Yes, the vs and vsimage check build failures were preventing this from being merged. I just kicked off a fresh set of tests with the comment above. Let's wait to see the results.

**Comment by ashishb-juniper** [comment]
@jleveque  : now only vsimage. Is there any way we can skip the vsimage test cases?

**Pull request merged**